### PR TITLE
Update /smart-displays page

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

**Waiting on the final +1 from marketing**
- Rename /digital-signage to /smart-displays
- Add redirection for /digital-signage
- Update /smart-displays page as per [copydoc](https://docs.google.com/document/d/1fFvBl-ZZvKDFV6OMv7O4kudQKtyxfvSU60rfVdKluAQ/edit#heading=h.oex3gsvjp5pm)
- Update other references (links, thank-you topics, etc) from digital-signage to smart-displays

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: https://ubuntu-com-11368.demos.haus/iot/smart-displays
    - Be sure to test on mobile, tablet and desktop screen sizes
- Compare against [copydoc](https://docs.google.com/document/d/1fFvBl-ZZvKDFV6OMv7O4kudQKtyxfvSU60rfVdKluAQ/edit#heading=h.oex3gsvjp5pm)
- Check that the redirects work ex. https://ubuntu-com-11368.demos.haus/iot/digital-signage more can be seen in the redirects.yaml changes

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4891

### CODE REVIEW

**Comment by spencerbygraves** [comment]
Hi @petesfrench, could you make a couple of changes please?

I think the core and frame logos could be a bit smaller, can you try 32px high please.

Also, I think the partners and boards sections would be better as 3 cards across, as below.

image

Thanks.

**Comment by spencerbygraves** [comment]
@petesfrench thanks for the updates 👍

File: templates/internet-of-things/smart-displays.html [hunk 1]
```diff
@@ -0,0 +1,337 @@
...
+
+{% block content %}
+
+<section class="p-strip--image is-dark is-deep u-no-padding--top u-no-padding--bottom" style="background-image: url(https://assets.ubuntu.com/v1/2c3ad93c-digital-signage-2.png); background-position: right bottom; background-size: 1246px auto;">
+  <div class="p-strip--suru-half-and-half-reversed">
+    <div class="row u-equal-height u-vertically-center">
+      <div class="col-6">
+        <h1>
+          Digital signage and kiosk <br class="u-hide--large">with Ubuntu
+        </h1>
```
Comment (minkyngkm) [inline templates/internet-of-things/smart-displays.html:17]: "The easy, secure, and reliable way of building smart displays for your business." is missing

```suggestion
        </h1>
        <p class="p-heading--4">The easy, secure, and reliable way of building smart displays for your business.</p>
```

File: templates/internet-of-things/smart-displays.html [hunk 2]
```diff
@@ -0,0 +1,337 @@
...
+            ) | safe
+          }}
+        </a>
+      </header>
+      <hr class="u-sv1 u-hide--small" />
+      <h3 class="p-card__title">4YouSee</h3>
+      <p class="p-card__content">A complete digital signage solution to create, manage and play your digital content, and analyse your audience.</p>
+    </div>
+    <div class="col-4 p-card">
+      <header class="no-border u-align--center u-vertically-center u-hide--small" style="min-height: 7rem;">
```
Comment (minkyngkm) [inline templates/internet-of-things/smart-displays.html:221]: should we need the `u-hide--medium`class? 
The image looks a bit on medium screen 
image


```suggestion
      <header class="no-border u-align--center u-vertically-center u-hide--small u-hide--medium" style="min-height: 7rem;">
```

**Comment by GAN-Canonical** [comment]
Thanks all for the nice work! Here is some feedback: 

1. Please, let's move the header image to the right so we can see the digital sign in the picture
image

2. This was the new section that we knew was going to be difficult to get right the first time. I think we are on the right path here. I have 3 main recommendations:
image

3. This section doesn't work for me anymore:

image

**Could we put this format instead:**
image

I will rewrite the section in the copy doc to fit this new format

thanks again!

**Comment by petesfrench** [comment]
Hey @minkyngkm, could you give this a QA as I have changed many links to the new url-address as well as changing the 'product' topic on some thank-you forms

File: templates/core/stories.html [hunk 3]
```diff
@@ -220,7 +220,7 @@ <h2>Ubuntu Core in digital signage</h2>
       <h3 class="p-heading--4">Screenly innovates IoT fleet management for secure digital signage</h3>
       <p>Originally having relied on home-grown infrastructure to deliver over-the-air (OTA) patches to devices in the field, Screenly needed a better way to manage its fast scaling and distributed IoT fleet. After a comprehensive market evaluation, Ubuntu Core stood out as the ideal fit.</p>
       <p>
-        <a href="/engage/secure-digital-signage-screenly">Screenly innovates IoT fleet management for secure digital signage&nbsp;&rsaquo;</a>
+        <a href="/engage/secure-smart-displays-screenly">Screenly innovates IoT fleet management for secure digital signage&nbsp;&rsaquo;</a>
```
Comment (minkyngkm) [inline templates/core/stories.html:223]: This page doesn't exist. Revert to `/engage/secure-digital-signage-screenly`?

Comment (GAN-Canonical) [reply templates/core/stories.html:223]: yes, that link is for the engage page of a case study: https://ubuntu.com/engage/secure-digital-signage-screenly
so we shouldn't change that URL

**Pull request merged**